backend/typescript/models/waitlistedCamper.model.ts

```diff
@@ -36,6 +37,12 @@ const WaitlistedCamperSchema: Schema = new Schema({
     type: String,
     required: true,
   },
+  status: {
+    type: String,
+    enum: ["NOT REGISTERED", "REGISTRATION FORM SENT", "REGISTERED"],
```

Comment:
nit: just to match other enums in our repo could we change these to camelcase? "NotRegistered", "RegistrationFormSent", "Registered"